spec/factories/image_works.rb

```diff
@@ -22,6 +22,13 @@
       end
     end
 
+    factory :image_work_with_one_metadata_file do
+      after(:create) do |image_work, evaluator|
+
+        1.times { image_work.ordered_members << FactoryGirl.create(:external_metadata_file, user: evaluator.user) }
```

Comment:
(from a non-programmer) this ordering aspect wouldn't have anything to do with constraints of the pcdm:hasMember predicate we're using to connect the various entities in the model, would it?